projection/ibc_channel/view/ibc_channels.go

```diff
@@ -430,10 +430,222 @@ func (ibcChannelsView *IBCChannels) List(
 	filter IBCChannelsListFilter,
 	pagination *pagination.Pagination,
 ) (
-	[]IBCChannelRow,
+	*[]IBCChannelRow,
```

Comment:
You don't need a pointer to slice if you just want to be able to return nil on error

Reply:
Good catch! Thanks! 

Here originally I want to avoid copy the return value... But I forgot slice is always a pointer... T_T